vendor/sigs.k8s.io/cluster-api-provider-azure/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
do we need this yet? Is go I can't find the Gopkg.* changes

Reply:
it was initially faked by adding a dummy implementation in the vendor.
it's dirty, but we can soon remove it.